shared/tsconfig.base.json

```diff
@@ -22,6 +22,7 @@
     "skipLibCheck": true,
     "strictNullChecks": true,
     "suppressImplicitAnyIndexErrors": true,
+    "ignoreDeprecations": "5.0",
```

Comment:
suppressImplicitAnyIndexErrors is deprecated and will be removed in TS5.5. We will need to refine our types in some places to remove this flag.

Reply:
https://sagebionetworks.jira.com/browse/PORTALS-2598